projects/components/src/tabs/content/tab-group.component.ts

```diff
@@ -31,6 +31,7 @@ import { TabComponent } from './tab/tab.component';
           <ng-template mat-tab-label>
             <div class="tab-label" [ngClass]="{ active: this.activeTabIndex === i }">
               {{ tab.label }}
+              <ht-info-icon [info]="tab.info"></ht-info-icon>
```

Comment:
ng-if

Reply:
This has been taken care of inside the `info-icon` component

Reply:
done